manifests/v/VSDC/Editor/7.1.1.390/VSDC.Editor.yaml

```
@@ -1,8 +1,8 @@
-# Created with YamlCreate.ps1 v2.1.0 $debug=NVS1.5-1-19041-1320
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
-
-PackageIdentifier: VSDC.Editor
-PackageVersion: 6.9.5.382
-DefaultLocale: en-US
-ManifestType: version
-ManifestVersion: 1.1.0
+# Created with YamlCreate.ps1 v2.1.2 $debug=AUSU.5-1-19041-1645
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: VSDC.Editor
+PackageVersion: 7.1.1.390

```

```suggestion
PackageVersion: 7.1.1.392
```
Version got updated.